djedi/backends/django/cache/backend.py

```diff
@@ -33,10 +32,14 @@ def _get_many(self, keys):
         return self._cache.get_many(keys)
 
     def _set(self, key, value):
-        self._cache.set(key, value, timeout=None)  # TODO: Fix eternal timeout like viewlet
+        self._cache.set(
+            key, value, timeout=None
+        )  # TODO: Fix eternal timeout like viewlet
```

Comment:
I'd prefer this to be kept as a one-liner and just move the comment ...

```suggestion
        # TODO: Fix eternal timeout like viewlet
        self._cache.set(key, value, timeout=None)
```

... same for the other places.